fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/AbstractMaterializedViewAggregateRule.java

```diff
@@ -60,15 +68,35 @@
  */
 public abstract class AbstractMaterializedViewAggregateRule extends AbstractMaterializedViewRule {
 
-    protected static final Map<Expression, Expression>
-            AGGREGATE_ROLL_UP_EQUIVALENT_FUNCTION_MAP = new HashMap<>();
+    // we only support roll up function which has only one argument currently
+    protected static final List<Pair<Expression, Expression>>
+            AGGREGATE_ROLL_UP_EQUIVALENT_FUNCTION_LIST = new ArrayList<>();
+    protected static final List<Expression> SUPPORTED_ROLL_UP_FUNCTIONS = new ArrayList<>();
```

Comment:
we should add a trait to let agg function impl it

Reply:
yeah, you are right. fix it with `aggregateFunction instanceof CouldRollUp`